docs/instances.md

```diff
@@ -203,15 +203,15 @@ The three `Process` entries above match directly to the three `Processes` items
 
 Inside each VM we delegate to [Monit](https://mmonit.com/monit/) to start, stop, and monitor the health of processes.
 
-Monit plays a small but vital role within every BOSH deployment running on Linux servers. Monit has been the process monitoring heart of BOSH instances since before BOSH was publicly open sourced in 2012. And ever since 2012, every single Product Manager of BOSH has said, "We will replace Monit with something else."
+Monit plays a small, but vital role within every BOSH deployment running on Linux servers. Monit has been the process monitoring heart of BOSH instances since before BOSH was publicly open sourced in 2012. And ever since 2012, every single Product Manager of BOSH has said, "We will replace Monit with something else."
```

Comment:
cut that. comma only comes before but if it splits two independent clauses.

Reply:
Done